mesa/space.py

```diff
@@ -610,3 +614,56 @@ def out_of_bounds(self, pos):
         x, y = pos
         return (x < self.x_min or x >= self.x_max or
                 y < self.y_min or y >= self.y_max)
+
+
+class NetworkGrid:
+    """ Network Grid where each node contains one agente. """
```

Comment:
@cauemello can you turn this into nodes as spaces that can contain more than one agent, so that if agents are traveling the grid from node to node and they meet at a node, then they interact?

Reply:
So agents would have a node_id location. That makes everything abstracted one more level away from the node... ie agent.node.get_neighbors to get the neighbors.